Jenkinsfile

```diff
@@ -92,6 +92,22 @@ pipeline {
       }
     }
 
+    stage("Soak Docker Release") {
+      agent {
+        label "mesos-ubuntu"
+      }
+      steps {
+        withCredentials([
+          string(credentialsId: '7bdd2775-2911-41ba-918f-59c8ae52326d', variable: 'DOCKER_HUB_USERNAME'),
+          string(credentialsId: '42f2e3fb-3f4f-47b2-a128-10ac6d0f6825', variable: 'DOCKER_HUB_PASSWORD')
+        ]) {
+          sh "docker --help"
```

Comment:
we don't really need this, do we?

Reply:
We do as long as I debug why we can not push to dockerhub :D